spring-cloud-stream-module-launcher/src/main/java/org/springframework/boot/loader/ModuleJarLauncher.java

```diff
@@ -45,8 +54,51 @@ protected void postProcessClassPathArchives(List<Archive> archives) throws Excep
 		archives.add(0, getArchive());
 	}
 
+
 	@Override
 	public void launch(String[] args) {
 		super.launch(args);
 	}
+
+	@Override
+	protected ClassLoader createClassLoader(URL[] urls) throws Exception {
+		Set<URL> copy = new LinkedHashSet<URL>(urls.length);
+		ClassLoader loader = getDefaultClassLoader();
+		if (loader instanceof URLClassLoader) {
+			for (URL url : ((URLClassLoader) loader).getURLs()) {
+				if (addDefaultClassloaderUrl(urls, url)) {
+					copy.add(url);
+				}
+			}
+		}
+		Collections.addAll(copy, urls);
+		return new LaunchedURLClassLoader(copy.toArray(new URL[copy.size()]), null);
+	}
+
+
+	private boolean addDefaultClassloaderUrl(URL[] urls, URL url) {
```

Comment:
if that's copied, I'll keep it as-is... hopefully this subclass will be temporary